velox/common/base/BitUtil.h

```diff
@@ -660,6 +660,20 @@ inline uint64_t hashMix(const uint64_t upper, const uint64_t lower) noexcept {
   return b;
 }
 
+// Order-independent way to reduce multiple 64 bit hashes into a
+// single hash. Copied from folly/hash/Hash.h because this is not
+// defined in some versions of folly.
+#if defined(FOLLY_DISABLE_UNDEFINED_BEHAVIOR_SANITIZER)
+FOLLY_DISABLE_UNDEFINED_BEHAVIOR_SANITIZER("unsigned-integer-overflow")
+#endif
+constexpr uint64_t commutativeHashMix(
```

Comment:
Lost comment?

----
- What's the motivation for making this function as "constexpr" instead of "inline"? Preexisting hashMix function is similar, but uses "inline".
- It would be nice to add a test for this function somewhere in BitUtilTest.cpp